docs/getting_started/prepare_environment.md

```diff
@@ -15,7 +15,7 @@ sudo add-apt-repository ppa:longsleep/golang-backports
 sudo apt-get update
 
 # Install required dependencies.
-sudo apt -y install git make tar wget curl rpm qemu-utils golang-1.17-go genisoimage python-minimal bison gawk
+sudo apt -y install git make tar wget curl rpm qemu-utils golang-1.17-go genisoimage python2-minimal bison gawk
```

Comment:
Definitely don't want python2.  What version of Ubuntu are you running on?

The dependencies are based on Ubuntu 18.04.  I believe there are some differences for Ubuntu 20.04/22.04.

You can probably substitute "python" for "python-minimal"

Here are the dependencies for officially building Mariner: https://github.com/microsoft/CBL-Mariner/blob/2.0/toolkit/docs/building/prerequisites.md

Reply:
I was running on 20.04 and changed this file to state that these tutorials have been validated on 20.04. I will replace "python-minimal" with "python", will that still work for 18.04?